lib/start_game.rb

```diff
@@ -4,7 +4,7 @@ def initialize(cpu_board, player_board)
   end
 
   def start
-    intro
+    puts "\nWelcome to BATTLESHIP\nEnter 'p' to play. Enter 'q' to quit."
```

Comment:
Removed the intro method because there was no need for a separate method to do this.